test/helpers/getSymbolDescription.js

```diff
@@ -39,12 +40,22 @@ test('getSymbolDescription', function (t) {
 			function (pair) {
 				var sym = pair[0];
 				var desc = pair[1];
-				st.equal(getSymbolDescription(sym), desc, debug(sym) + ' yields ' + debug(desc));
+				st.equal(getSymbolDescription(sym), desc, debug(sym) + ' description is ' + debug(desc));
 			}
 		);
 
-		st.test('only possible when inference is supported', { skip: !getInferredName }, function (s2t) {
-			s2t.equal(getSymbolDescription(Symbol('')), '', 'Symbol("") description is empty string');
+		st.test('only possible when inference or native `Symbol.prototype.description` is supported', {
+			skip: !getInferredName && !has(Symbol.prototype, 'description')
```

Comment:
why would we need to check this case? Any environment that has a native Symbol.prototype.description getter will already support inference, so the test should Just Work.

Reply:
Inference will not work when `eval` is disabled.

Reply:
Sure, but the tests aren't ran in any environment where eval is disabled.